docs/source/theory/intro.rst

```diff
@@ -48,6 +48,19 @@ This is an overview of physical assumptions implemented in the numerics of Impac
 Tracking and Lattice Optics
 """""""""""""""""""""""""""
 
+Tracking through lattice optics in ImpactX is performed by updating the canonical phase space variables (x,px,y,py,t,pt) using symplectic transport.
+The elements supported currently fall into one of three categories:
+
+* **ideal (thick) elements** using a hard-edge fringe field approximation, such as drifts, quadrupoles, and dipoles
```

Comment:
Should we order this also by approximation?
- thin
- thick
- soft-edge
?

Reply:
OK, yes it makes sense to order them this way.  I'm not sure about adding the term "approximation" explicitly, because right now these categories apply to completely different elements.  Also, we might want to add additional categories (eg, for surrogate models or "black box" elements).

Reply:
Agree, yes I used approximation above in lieu of a better word :sweat_smile:

Reply:
> Also, we might want to add additional categories (eg, for surrogate models or "black box" elements).

Good point, will add the docs for them here then as well for a follow-up :)
We could use the term "magic box" though, which consequently makes us wizards :magic_wand: .

Reply:
Done.